code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const LoadingIndicator = () => {
+  return (
+    <>
+      <div>
+        <p>Loading!!!</p>
+      </div>
+    </>
```

Comment:
You have only one immediate child element (div) so you shouldn't use react fragment here.